az-digital/az_quickstart
# Closes #1694: Add granular class selector to search block Submit button

The submit box on the search bar in the main nav was using an obscure CSS selector, so I added `az-search-block` class and applied it to the main CSS sheet. This change should maintain existing functionality and give us a way to style that block specifically without breaking other placements.

## Description
Added class `az-search-block` to style.css and also to the Search block using the Block Configure tool (/admin/structure/block/manage/az_barrio_search?destination=/admin/structure/block)

## Related issues


## How to test
Go to arizona.edu homepage and search. When it takes you to the Search page, search again. The submit button should be white on pageload, and red when you hover.

Off canvas considerations.

## Types of changes
Added new CSS class to existing entry. Applied that class using the Drupal blocks UI.

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
